data-integration/README.md

```diff
@@ -0,0 +1,315 @@
+        }
+        return http:CREATED;
+    }
+
+    resource function get task/[int taskId]() returns store:Task|http:NotFound|http:InternalServerError {
+        store:Task|persist:Error task = dbClient->/tasks/[taskId];
+        if task is persist:NotFoundError {
+            return http:NOT_FOUND;
+        }
+        if task is persist:Error {
+            return <http:InternalServerError>{body: task.message()};
+        }
+        return task;
+    }
+
+    resource function get employeetasks/[int empId]() returns store:Task[]|http:NotFound|http:InternalServerError {
+        store:Task[]|persist:Error tasks = from store:Task task in dbClient->/tasks(store:Task)
+            where task.employeeId == empId
+            select task;
+        if tasks is persist:NotFoundError {
+            return http:NOT_FOUND;
+        }
+        if tasks is persist:Error {
+            return <http:InternalServerError>{body: tasks.message()};
+        }
+        return tasks;
+    }
+
+    resource function get employee/[int empId]() returns store:Employee|http:NotFound|http:InternalServerError {
+        store:Employee|persist:Error employee = dbClient->/employees/[empId];
+        if employee is persist:NotFoundError {
+            return http:NOT_FOUND;
+        }
+        if employee is persist:Error {
+            return <http:InternalServerError>{body: employee.message()};
+        }
+        return employee;
+    }
+
+    resource function put task/[int taskId](store:TaskUpdate task)
+            returns store:Task|http:InternalServerError {
+        store:Task|persist:Error updatedTask = dbClient->/tasks/[taskId].put(task);
+        if updatedTask is persist:Error {
+            return <http:InternalServerError>{body: updatedTask.message()};
+        }
+        return updatedTask;
+    }
+
+    resource function delete task/[int taskId]()
+            returns http:NoContent|http:NotFound|http:InternalServerError {
+        store:Task|persist:Error task = dbClient->/tasks/[taskId];
+        if task is persist:NotFoundError {
+            return http:NOT_FOUND;
+        }
+        if task is persist:Error {
+            return <http:InternalServerError>{body: task.message()};
+        }
+        if task.status != store:COMPLETED {
+            return <http:InternalServerError>{body: "Task is not completed yet"};
+        }
+        store:Task|persist:Error deleteResult = dbClient->/tasks/[taskId].delete;
+        if deleteResult is persist:Error {
+            return <http:InternalServerError>{body: deleteResult.message()};
+        }
+        return http:NO_CONTENT;
+    }
+}
+```
+
+#### Entity Relationship Diagram
+
+The [entity relationship diagram view](https://wso2.com/ballerina/vscode/docs/implement-the-code/sequence-diagram-view/) for the defined data model is the following.
+
+<img src="./resources/entity_diagram.png" alt="Entity Diagram" height="800" style="width:auto; max-width:100%">
+
+### Step 3: Start the docker service
```

Comment:
Is this for the databse server? if so let's ask the user to make sure that the databse server runs